cas/cloudcas/cloudcas_test.go

```diff
@@ -673,3 +775,445 @@ func Test_getCertificateAndChain(t *testing.T) {
 		})
 	}
 }
+
+func TestCloudCAS_CreateCertificateAuthority(t *testing.T) {
+	must := func(a, b interface{}) interface{} {
+		return a
+	}
+
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	mosCtrl := gomock.NewController(t)
+	defer mosCtrl.Finish()
+
+	m := NewMockCertificateAuthorityClient(ctrl)
+	mos := NewMockOperationsServer(mosCtrl)
```

Comment:
You need to write at least one expectation per function, it will fail if one is missing, but you can also write an expectation as a function to return different things depending on the input and allow it to be used multiple times:
https://pkg.go.dev/github.com/golang/mock/gomock#Call.Do
https://pkg.go.dev/github.com/golang/mock/gomock#Call.MaxTimes